src/test/tasks.yaml

```diff
@@ -57,6 +57,37 @@ tasks:
             address: demo-8081.uds.dev
             code: 200
 
+      - description: Verify the authservice tenant app is accessible
+        wait:
+          network:
+            protocol: https
+            address: protected.uds.dev
+            code: 200
+
+      - description: Wait for authservice to reload configuration
+        wait:
+          cluster:
+            kind: Deployment
+            name: authservice
+            namespace: authservice
+
+      - description: Verify the authservice tenant app is protected by checking redirect
+        maxRetries: 3
+        cmd: |
+          set -e
+          SSO_REDIRECT=$(uds zarf tools kubectl run curl-test --image=cgr.dev/chainguard/curl:latest -q --restart=Never --rm -i -- -Ls -o /dev/null -w %{url_effective} "https://protected.uds.dev")
```

Comment:
I did this because I did not want to have to ensure curl existed when the tasks run.  Happy to conform as it looks like its already required but my thought was that this test package + validate would be more portable.